.github/workflows/build.yml

```diff
@@ -36,3 +36,7 @@ jobs:
       with:
         name: dist-Release-front_end
         path: src/out/Release/gen/front_end
+    - name: Run unit tests
+      run: |
+        cd src
+        npm run unittest -- --expanded-reportin
```

Comment:
expanded-reporting
G in the end is missing :)

Reply:
Both works for some reason lol. I guess Karma is resilient to typos haha

<img width="1399" alt="screenshot showing the command running successfully with the flag '--expanded-reportin' with a missing 'g'" src="https://github.com/user-attachments/assets/d3281f99-54b3-4fae-8e35-722c92677c22">

Added the `g` back.

![image](https://github.com/user-attachments/assets/19fcf34b-8e50-4f5c-851f-8e2f68e6cf46)
https://www.dictionary.com/e/slang/they-see-me-rollin/